collection/collectors.py

```diff
@@ -284,11 +284,11 @@ def collect(self) -> Generator[Any, Any, Any]:
                 prototypes = surface_force_data["prototypes"]
                 for prototype, prototype_data in prototypes.items():
                     force_consumption_stats.add_metric(
-                        labels=[force, prototype, surface, prototype_data["type"]],
```

Comment:
If we're removing the `type` label from the production/consumption metrics we should also remove it from the list of metric labels on lines `274` and `279`.